velox/exec/Spiller.h

```diff
@@ -39,8 +39,10 @@ class Spiller {
     kOrderByInput = 4,
     // Used for order by output processing stage.
     kOrderByOutput = 5,
+    // Used for row number.
+    kRowNumber = 6,
```

Comment:
It would be nice to move away from having all spilling logic from all operators be in a single class.